### DISCUSSION TITLE
New user to OsmAnd - how to import bulk favorite locations

### DISCUSSION THREAD

**Discussion by below130** [osmandapp/OsmAnd#11100] [Q&A]
I'm new to OsmAnd and am using it in an Android based head unit in my car.  So far very impressed!  I have a large number of locations that I'd like to input and wondering if there is a way to create a file of addresses and import it so I don't have to find each one on the map individually.  

Thanks in Advance
Dan

**Comment by thompehw** [comment]
Just until someone wiser comes along....

OsmAnd can import a GPX POI file into favourites, but that needs coordinates for the addresses. Just doing a quick search, there may be tools to turn a list of addresses into GPX POIs, e.g. this discussion: [http://www.poi-factory.com/node/8692](http://www.poi-factory.com/node/8692)

**Comment by below130** [comment]
This is some great information!  Thanks.  I will give this a try and see if I can make it work!

 

Thanks again
